libraries/classes/Table.php

```diff
@@ -2264,6 +2264,42 @@ public function getSqlQueryForIndexCreateOrEdit($index, &$error)
         return $sql_query;
     }
 
+    public function getSqlQueryForIndexRename($index, &$error){
```

Comment:
```suggestion
    public function getSqlQueryForIndexRename(PleaseFindRightClass $index, ?Message &$error): string {
```

Can you add a phpdoc block and be sure to run `composer run phpcs`/`phpcbf`

Reply:
@williamdes, By phpdoc block you mean information regarding the function and its working just above the function right?

Reply:
Yes the
```
/**
  *
  */
```
Block